doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+  the data will be removed after two weeks, so you will have to move the output data
+  to the `/work` partition, via e.g. a `nohup` job:
+
+  .. code-block:: bash
+
+    nohup cp -r /scratch/b/$USER/esmvaltool_output/* /work/bd0854/b382109/v2xx
+
+  where `bd0854/b382109` is the project location in `work`
+
+
+The `/work` partition is visible by the VM so you can run the compare tool straight on the VM.
+
+Do not store final release results on the VM including `/preproc/` dirs, the total
+size for all the recipes output, including `/preproc/` dirs is in the 4.5TB ballpark,
+much too high for the VM storage capacity!
+
+The steps to running the compare tool on the VM are the following:
+
+- run date: log the run date here
+- conda env: log the name of the conda environment you are using
+- ESMValTool branch: log the name of the code branch you are using (e.g. `v2.8.x`)
+- prerquisite - install `imagehash`: `pip install imagehash`
+- reference run (v2.7.0): `export reference_dir=/work/bd0854/b382109/v270` (contains `preproc/` dirs too, 122 recipes)
+- current run (v2.8.0): `export current_dir=path_to_current_run`
+- command to run: `nohup python ESMValTool/esmvaltool/utils/testing/regression/compare.py --reference $reference_dir --current $current_dir > compare_v280_output.txt`
+
+Some of the recipes will appear as having identical output to the one from previous release. However, others
+will need human inspection; here are some guidelines on how to perform the human inspection:
+
+- look at plots from current run vs previous release run: most of them will be identical, but if Matplotlib
+  has changed some plotting feature, images will have slightly different metadata so the comparison script will report them
+  as different - but Mark I eyeball inspection will show they are identical
+- other plots will differ due to changes in plot settings (different colours, axes etc) due to updated settings from the
+  diagnostic developers: if they look similar enough, then it's fine
+- report (and subsequently open issues) if you notice major differences in plots; most times a simple comment on the
+  release issue, whereby you tag the diagnostic developers leads to them having a look at the plots and OK-ing them; if that's
+  not the case, then open a separate issue
+
+Appendix
+--------
+
+Here you can find a list of useful files and directories:
+
+- Formatted list of current recipes (as of v2.7.0) to be used with Markdown entries (on DKRZ/Lvante) at `/home/b/b382109/Tool_Release_270_Scripts/all_recipes.md` or in the draft Pull Request https://github.com/ESMValGroup/ESMValTool/pull/2883
+- last release (v2.7.0) submit scripts on DKRZ/Levante `/home/b/b382109/submit`
+- Miniconda3 installer file on DKRZ/Levante `/home/b/b382109/Miniconda3-py39_4.12.0-Linux-x86_64.sh` (remember to immediately update conda after using it, it is fairly old, from May 2022)
+- list of Autoassess reference files and masks on DKRZ/Levante `/home/b/b382109/autoassess_files`
```

Comment:
This information seems prone to going out of date very quickly
- Our source code should be in a public GitHub repo
- Mamba installer can be downloaded from https://github.com/conda-forge/miniforge#mambaforge
- Data for running recipes should be publicly available. If that fails, it should at the very least be in a group workspace instead of someones home directory, but actually I would consider such recipes broken according to our broken recipe policy.